python-template/{{cookiecutter.placeholder_repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,8 @@
-**Description:** Describe in a couple of sentences what this PR adds
-
-**JIRA:** Link to JIRA ticket
-
-**Dependencies:** dependencies on other outstanding PRs, issues, etc. 
-
-**Merge deadline:** List merge deadline (if any)
-
-**Installation instructions:** List any non-trivial installation 
-instructions.
-
-**Testing instructions:**
-
-1. Open page A
-2. Do thing B
-3. Expect C to happen
-4. If D happened instead - check failed.
-
-**Reviewers:**
-- [ ] tag reviewer 
-- [ ] tag reviewer 
 
 **Merge checklist:**
-- [ ] All reviewers approved
-- [ ] CI build is green
+Check off if complete or not applicable:
 - [ ] Version bumped
 - [ ] Changelog record added
 - [ ] Documentation updated (not only docstrings)
 - [ ] Commits are squashed
```

Comment:
```suggestion
- [ ] Commits are squashed into one or more [conventional commits](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html)
```

Reply:
Oh, Phil raised an interesting question in chat -- we already have a commitlint CI check. Still worth the extra text? Could go either way.